matsim/scenariogen/data/__init__.py

```diff
@@ -327,16 +332,24 @@ class Trip:
     purpose: Purpose
     sd_group: SourceDestinationGroup
     valid: bool
+    dep_district: str = ""
```

Comment:
If would rather use None as default to indicate this information is not available.